packages/cdk-graph-plugin-diagram/src/config.ts

```diff
@@ -51,6 +52,9 @@ export interface IDiagramConfigBase {
 
   /** Config for graph theme */
   readonly theme?: GraphThemeConfigProp;
+
+  /** Config for predetermined node positions */
+  readonly nodePositions?: Map<string, Point>;
```

Comment:
Looks like the build is failing because JSII doesn't like `Map` - I think you should be able to do an object type like: `{ [nodeId: string]: Point }`

Also I think we might need to define our own `Point` interface as I don't think ts-graphviz uses JSII :)